node/src/sub_lib/neighborhood.rs

```diff
@@ -381,6 +381,7 @@ pub struct NeighborhoodSubs {
     pub bind: Recipient<BindMessage>,
     pub start: Recipient<StartMessage>,
     pub new_public_ip: Recipient<NewPublicIp>,
+    #[cfg(test)]
```

Comment:
If we could use `AssertionMessage` (or whatever it's called) to perform the functions now handled by `NodeQueryMessage`, we could get rid of `NodeQueryMessage` and all its special-casing entirely.